src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -161,6 +208,8 @@ private void Request(ApplicationEngine engine, string url, string filter, string
             if (list.Count >= 256)
                 throw new InvalidOperationException("There are too many pending responses for this url");
             list.Add(id);
+
+            engine.SendNotification(Hash, "Request", new VM.Types.Array { id, url, filter });
```

Comment:
Maybe this event's can be done by the project.

Reply:
?